c2corg_api/models/document.py

```diff
@@ -97,12 +202,38 @@ def document_id(self):
 class DocumentLocale(Base, _DocumentLocaleMixin):
     __tablename__ = 'documents_locales'
 
-    _ATTRIBUTES = ['document_id', 'culture', 'title', 'description']
+    __mapper_args__ = {
+        'polymorphic_identity': 'd',
+        'polymorphic_on': _DocumentLocaleMixin.type,
+        'version_id_col': _DocumentLocaleMixin.version,
+        'version_id_generator': lambda version: uuid.uuid4().hex
```

Comment:
I was wondering about the use of `uuid4` versus `uuid1`. Using `uuid4` should be safe. See http://stackoverflow.com/questions/703035/when-are-you-truly-forced-to-use-uuid-as-part-of-the-design/786541#786541 for an interesting SO answer on that subject.